src/args.rs

```diff
@@ -0,0 +1,37 @@
+use clap::Parser;
+
+#[derive(Parser, Debug)]
+#[clap(author, version, about = "A pomodoro timer CLI app")]
+pub struct PomodoroArgs {
+  #[clap(
```

Comment:
Does `command` work? Any reasons to use `clap` vs `command`?

Reply:
mainly for default arguments